pyvista/plotting/plotting.py

```diff
@@ -2083,6 +2101,11 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
         cmap = kwargs.pop('colormap', cmap)
         culling = kwargs.pop("backface_culling", culling)
 
+        # account for legacy behavior
+        if 'stitle' in kwargs:  # pragma: no cover
+            warnings.warn(USE_SCALAR_BAR_ARGS)
```

Comment:
Same as above regarding `DeprecationWarning`.

Reply:
Fixed in d0846067a565bc6eed0cd4fc5517ef2629c559d9.